### ISSUE TITLE
Animation issue or not?

### ISSUE THREAD

**Issue by jpdeglet69** [minorua/Qgis2threejs#320]
Hello,

I define an animation (only images),
I select "start animation once..." 
When I run, the animation stops on each images, I need to click or enter to continue the animation.

To test:
https://eleana.online/@DigitalTwins/QGIS/KONGO/NGungi/Narrative/index.html

Is-it possible to customize a complete automatic animation?

Last version of Chrome on Mac OS.

I'm a beginner under QGIS but your plugin, really nice for my needs!!!

BR.
JPhD

**Comment by JoonyDee** [comment]
Yes, you can: just insert some frames without narrative content; right-click on keyframe and adjust "easing". 

BTW: how did you generate the appearence of the Layer NGungi survey 730 3D (it is Polygon, but how did you get the x-y-z coords?

**Comment by jpdeglet69** [comment]
> BTW: how did you generate the appearence of the Layer NGungi survey 730 3D (it is Polygon, but how did you get the x-y-z coords?

Hi,
By topo survey on site then processing the informations in topography software for caving and export results to shape file.
Below an example
<img width="936" alt="Capture d’écran 2024-02-15 à 15 42 15" src="https://github.com/minorua/Qgis2threejs/assets/36116259/f86ed634-8649-4cc9-bf1f-4675c68219e7">
BR
JPhD

**Comment by jpdeglet69** [comment]
> Yes, you can: just insert some frames without narrative content; right-click on keyframe and adjust "easing".

My goal is to produce an automatic clip with narrative content.
BR
